src/shared/components/Dropzone/index.js

```diff
@@ -74,6 +76,38 @@ const Dropzone = ({
     bottom: wrapperHeight.current - (headHeight + (rowNumber + 1) * rowHeight),
   } : {};
 
+  const getSvg = (overrideSvgProps) => {
+    const defaultHeight = Math.max(objectsList.length * rowHeight, wrapperHeight.current);
+
+    return (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width={`calc(100% - ${RAINBOW_BORDER_WIDTH}px)`}
+        height={`${defaultHeight + RAINBOW_BORDER_WIDTH / 2}px`}
+        className={classes.rainbowField}
+      >
+        <linearGradient id="gradient" x1="0%" y1="25%" x2="100%" y2="75%">
```

Comment:
Can we have an id field that is less generic? Maybe something like `gradient-backdrop`.
For example, on the space homepage I'm remember having problems with SVGs because some SVGs had the same id as other SVGs and it caused problems with the styling.

Reply:
Yea, you are totally right, thanks!

Reply:
fixed 29436943ffdad65567def699d98acd527e1db8ff ad9243dff9c7de8c17bb4600fa3afb7628550481

Reply:
LGTM